demo/monitoring-all-process-instances/js/execution-data/execution-data.js

```diff
@@ -198,8 +199,49 @@ class ExecutionData {
             }
         };
         if (pathClass) {
-            data.pathClass = pathClass;
+            data.styleUpdate = buildStyleUpdateOptions(pathClass);
```

Comment:
**suggestion:** 
I thought that the property and the functions for the style of the path were not clear. 
It took me a while to understand what they were referring to 🙁

Reply:
Let's rename
* styleUpdate --> pathStyle
* buildStyleUpdateOptions --> buildPathStyle
* pathClass variable: find a name